server/middleware/insertJtiIntoDB.js

```diff
@@ -5,7 +5,7 @@ const logger = require('../../config/logger');
 const insertJtiIntoDB = (request, response, next) => {
 	const { userId } = request.body;
 	const jti = uuidv4();
-	const TEN_MINS_IN_SECONDS = 10 * 60;
+	const TEN_MINS_IN_SECONDS = 20 * 60;
```

Comment:
I don't think this is ten minutes any more ;)

Reply:
If you have a config/constants file, it might make sense to move this there, and rename it "jwtExpireTimeSeconds".... or something. Give it a better name than what I suggested.